_sass/landing-page.scss

```diff
@@ -469,8 +469,9 @@
 .getting-started {
   color: #ffffff;
   text-align: center;
-  background: var(--background-grey);
+  // background: var(--background-grey);
   // background: linear-gradient(to top, $menu-color -80%, $primary-brand-color 80%) no-repeat #a05fb7;
+  background: linear-gradient(to right bottom, #000000,#000000, #00483c);
```

Comment:
we don't define colors like this, it is defined like variables in root.scss and then used